### PR TITLE
Feature/style guide and minor related fixes

### PR DESCRIPTION
**Pull request by saadshahd**
Hi, Dr. تقبل مروري :smile_cat: 
- [x] In lieu of maintaining the existing coding style, used a formal Style-guides :dancers:.
- [x] Used the array methods(forEach, every) instead of the classic for loops.
- [x] Add some consistency  to the code by unifying the strings quote to be `'`.
- [x] Fix the indentation to meet the introduced Editorconfig rules.

### CODE REVIEW

**Comment by drdrsh** [comment]
شكرا يا سيدي عالشغل :+1: 

Still, I have some concerns.
1. I don't like the one line if statements. I reckon they are part of the "airbnb style guide" right?. I always prefer to use curly braces even if I don't have to. It is verbose and adds more lines but it is much less error prone and a good habit to pick up and maintain. I prefer [Khan Academy](https://github.com/Khan/style-guides/blob/master/style/javascript.md) style guide, even though I am guilty of not sticking to it. :disappointed: 
2.  for loops outperform forEach() and every() as shown [here](http://jsperf.com/for-vs-foreach/66), I am okay with the modifications as the performance hit isn't all that important for this weekend plugin but I thought I'd point that out.

**Comment by saadshahd** [comment]
العفو :kissing:

I agree with your concerns, I have one explanation for the one line if statement. and the other concern actually thanks for pointing that. I haven't benchmarked those before, it's nice to know the results.

The explanation: [The rule](https://github.com/airbnb/javascript/#blocks--braces)

I think it's okay to use one-line if statement while It's not written like this:

```
if (a)
  foo();
```

because this formula will cause a troubles for the future developers when they do:

```
if (a)
  foo();
  bar();
```

they will expect `foo` and `bar` to be invoked but only `foo` does.

But if we wrote it from the beginning as:
`if (a) foo();`

the future developers will do:

```
if (a) {
  foo();
  bar();
}
```

Is this a good argument ?
ولا أحزن في نفسي :cry:

**Comment by drdrsh** [comment]
tl;dr "احزن في نفسك" :laughing: 

I understand your point. But I viewed airbnb guide and they don't seem to have a problem with block braces that contain one line. So my code is not in violation of the guide. Also, I am saving the future developer the 3 extra characters =).

I like how neat and clean a one line if statement looks like. But In most cases I value verbosity and expressiveness of code over brevity. For that reason I am okay with the forEach() and every() changes you introduced.

**Comment by saadshahd** [comment]
:smile: ديكتاتور
Updated.
